mk/versions.mk

```
@@ -42,7 +42,7 @@ test-$(1)::
 	@echo "   $(2)_VERSION=$($(2)_VERSION)"
 	@echo "   $(2)_PATH=$($(2)_PATH) => $(abspath $($(2)_PATH))"
 
-reset-$(1)::
+reset-$(1):: net6.config
```

This isn't needed, Make will automatically make included files, so this line from above has the same effect:

```make
-include $(TOP)/net6.config
```

```suggestion
reset-$(1)::
```